pkg/apis/core/types.go

```diff
@@ -1774,7 +1774,7 @@ type EnvVar struct {
 // EnvVarSource represents a source for the value of an EnvVar.
 // Only one of its fields may be set.
 type EnvVarSource struct {
-	// Selects a field of the pod: supports metadata.name, metadata.namespace, metadata.labels, metadata.annotations,
+	// Selects a field of the pod: supports metadata.name, metadata.namespace, metadata.labels['<KEY>'], metadata.annotations['<KEY>'],
```

Comment:
Since this comment goes to the swagger json and then translates to HTML output in the API reference, we should avoid using characters like "<" or ">". So a better revision would be:

```suggestion
	// Selects a field of the pod: supports `metadata.name`, `metadata.namespace`, `metadata.labels['<KEY>']`, metadata.annotations['<KEY>'],
```

Reply:
Thank you very much for your reminder, the change has been completed ~